php/libraries/NDB_BVL_Battery.class.inc

```diff
@@ -478,6 +478,10 @@ class NDB_BVL_Battery extends PEAR
             $query .= " AND b.CenterID IS NULL";
         }
 
+        if ($firstVisit === false) {
+            $query .= " AND (b.firstVisit IS NULL OR b.firstVisit='N')";
+        }
```

Comment:
Is this a bug in the Battery class? Or something to do with the tests you are writing?